ext/random/random.c

```diff
@@ -119,6 +119,7 @@ static inline uint32_t rand_range32(const php_random_algo *algo, php_random_stat
 	while (UNEXPECTED(result > limit)) {
 		/* If the requirements cannot be met in a cycles, return fail */
 		if (++count > 50) {
+			zend_throw_exception(spl_ce_RuntimeException, "Failed to generate an acceptable random number in 50 attempts", 0);
```

Comment:
This feels more like it should just be a plain ``Error`` then anything else

Reply:
Ditto, an `Error` thrown here could mess things up...